Source/Core/InputCommon/ControllerInterface/DualShockUDPClient/DualShockUDPClient.cpp

```diff
@@ -342,19 +340,25 @@ Device::Device(Proto::DsModel model, int index) : m_model{model}, m_index{index}
   AddInput(new TouchInput("Touch Y-", m_touch_y, -TOUCH_Y_AXIS_MAX));
   AddInput(new TouchInput("Touch Y+", m_touch_y, TOUCH_Y_AXIS_MAX));
 
-  AddInput(new AccelerometerInput("Accel Left", m_accel.x, 1));
-  AddInput(new AccelerometerInput("Accel Right", m_accel.x, -1));
-  AddInput(new AccelerometerInput("Accel Backward", m_accel.y, 1));
-  AddInput(new AccelerometerInput("Accel Forward", m_accel.y, -1));
-  AddInput(new AccelerometerInput("Accel Up", m_accel.z, 1));
-  AddInput(new AccelerometerInput("Accel Down", m_accel.z, -1));
-
-  AddInput(new GyroInput("Gyro Pitch Up", m_gyro.x, -1));
-  AddInput(new GyroInput("Gyro Pitch Down", m_gyro.x, 1));
-  AddInput(new GyroInput("Gyro Roll Right", m_gyro.y, -1));
-  AddInput(new GyroInput("Gyro Roll Left", m_gyro.y, 1));
-  AddInput(new GyroInput("Gyro Yaw Right", m_gyro.z, -1));
-  AddInput(new GyroInput("Gyro Yaw Left", m_gyro.z, 1));
+  // Convert Gs to meters per second squared
+  constexpr auto accel_scale = 1.0 / GRAVITY_ACCELERATION;
+
+  AddInput(new AccelerometerInput("Accel Left", m_pad_data.accelerometer_x_g, accel_scale));
+  AddInput(new AccelerometerInput("Accel Right", m_pad_data.accelerometer_x_g, -accel_scale));
+  AddInput(new AccelerometerInput("Accel Up", m_pad_data.accelerometer_y_g, -accel_scale));
+  AddInput(new AccelerometerInput("Accel Down", m_pad_data.accelerometer_y_g, accel_scale));
+  AddInput(new AccelerometerInput("Accel Backward", m_pad_data.accelerometer_z_g, -accel_scale));
+  AddInput(new AccelerometerInput("Accel Forward", m_pad_data.accelerometer_z_g, accel_scale));
+
+  // Convert degrees per second to radians per second
+  constexpr auto gyro_scale = 360.0 / MathUtil::TAU;
```

Comment:
Guessing this is swapped since we divide by the range in `GetState`?

Reply:
Yes.